Appraisals

```diff
@@ -1,5 +1,25 @@
 # frozen_string_literal: true
 
+appraise "nokogiri-18.8" do
```

Comment:
let's name these nokogiri-1.18 (with the major, but without the patch). it's confusing to skip the major version, and we by omitting the patch version we make it clearer that we're testing the latest patch version within that minor version. In fact, to make that clearer, let's also change the requirement to `gem "nokogiri", "~> 1.18.8"`

Reply:
Changed in https://github.com/instructure/nokogiri-xmlsec-instructure/pull/21/commits/3f6887a64bc06af87c21471b15d7335394b63c93